instrumented/nightly-tests/src/androidTest/kotlin/com/datadog/android/nightly/logs/LogsE2ETests.kt

```diff
@@ -0,0 +1,262 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.logs
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import com.datadog.android.log.Logger
+import com.datadog.android.nightly.TEST_METHOD_NAME_KEY
+import com.datadog.android.nightly.utils.measure
+import com.datadog.tools.unit.forge.aThrowable
+import fr.xgouchet.elmyr.junit4.ForgeRule
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+internal class LogsE2ETests {
+
+    @get:Rule
+    val forge = ForgeRule()
+
+    lateinit var logger: Logger
+
+    @Before
+    fun setUp() {
+        logger = Logger.Builder()
+            .setLoggerName(LOGGER_NAME)
+            .build()
+    }
+
+    /**
+     * apiMethodSignature: Logger#fun v(String, Throwable? = null, Map<String, Any?> = emptyMap())
+     */
+    @Test
+    fun send_a_verbose_log() {
```

Comment:
This name feels slightly unsound to me (using a natural language syntax, we're creating a log, not sending it. Also we could create a log from a span in the future, shouldn't we disambiguate that). I'd feel more comfortable with a test name that had some kind of link with the API under test. 

Also I'm not sure how the naming will scale, and i'd like each test name to be: 
 - consistent
 - easy to generate for future tests
 - linked with the api under test
 
I didn't see in your RFC a naming convention, but maybe we should define one first to ensure all future tests will be easy to name ?

Reply:
Agree, let me add a proposal in the RFC and we can debate there.